### ISSUE TITLE
Is it available with esp32 with wifi connection?

### ISSUE THREAD

**Issue by hemangjoshi37a** [Neumi/ethersweep#28]
Is it available with esp32 with wifi connection?

**Comment by Neumi** [comment]
Currently not. There might be a simplified version with WiFi in the future. 

If you have a good approach to handling the password and wifi credential settings, let me know!

**Comment by hemangjoshi37a** [comment]
you  can use wifi manager https://github.com/tzapu/WiFiManager

**Comment by Neumi** [comment]
@hemangjoshi37a yes, that would probably be a good approach. But I don't like the manual work if you need more than a few motors. But this could indeed be ok for smaller setups.
